src/components/validations/with-validation.spec.js

```diff
@@ -386,25 +384,8 @@ describe('when the withValidations HOC wraps a component', () => {
     });
   });
 
-  describe('get validation type', () => {
-    const valTypes = ['error', 'warning', 'info', 'empty'];
-
-    describe.each(valTypes)('%s', (type) => {
-      it(`type should be ${type}`, () => {
-        const obj = {
-          hasError: false,
-          hasWarning: false,
-          hasInfo: false
-        };
-        const typeToKey = `has${text.titleCase(type)}`;
-        const value = type === 'empty' ? '' : type;
-
-        if (Object.prototype.hasOwnProperty.call(obj, typeToKey)) {
-          obj[typeToKey] = true;
-        }
-
-        expect(getValidationType(obj)).toEqual(value);
-      });
-    });
+  it('does not invoke components onChange if it is not passed as a prop', () => {
```

Comment:
This negative assertion feels weird. I guess it's trying to catch attempting to call `undefined` as a function.

Reply:
I've added it just for the sake of having 100% coverage.
I'm planning to remove all validation leftovers in another PR where I'll remove both `Form` components from `__experimental__` and `__deprecated__`